lib/react/rails/railtie.rb

```diff
@@ -83,6 +83,10 @@ class Railtie < ::Rails::Railtie
         React::ServerRendering.reset_pool
         # Reload renderers in dev when files change
         ActionDispatch::Reloader.to_prepare { React::ServerRendering.reset_pool }
+        if defined? RSpec::Rails
+          require 'react/rails/test_helper'
```

Comment:
Would it hurt to _always_ require this file? It's not Rspec-specific, right? So we could move it to `react/rails.rb` along with the other `require` calls. 

Also, if this helper is specific to Rspec, what do you think about calling it `RspecHelper` instead of `TestHelper`? I think that expresses its purpose a little bit more.